server/catgenome/src/test/java/com/epam/catgenome/util/TabixReaderTest.java

```diff
@@ -0,0 +1,173 @@
+package com.epam.catgenome.util;
+
+import com.epam.catgenome.util.feature.reader.TabixIteratorLineReader;
+import com.epam.catgenome.util.feature.reader.TabixReader;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration({"classpath:applicationContext-test.xml"})
+public class TabixReaderTest {
+
+    static String tabixFilePath = "classpath:templates/trioDup.vcf.gz";
```

Comment:
Constants should be final and should follow CONSTANT_NAMING_TEMPLATE. Also, better declare everything private, if you have no intent to use it anywhere else.